products/health-care/appointments/va-online-scheduling/research/july-2020-user-research/vaosr-june-research-plan.md

```diff
@@ -1,50 +1,86 @@
+   8. What are veterans expectations for telehealth/video appointments through VAOSR? What are veterans issues for telehealth/video appointments through VAOSR?
+   9. Do Veterans hit certain roadblocks in their scheduling experience in new VAOSR? What are those roadblocks?
+   10. What, if any, messaging options do veterans expect to have available to them while scheduling an appointment online? Who do veterans expect to be able to message with in this scenario?
 4. Hypothesis: What is your hypothesis for this research? 	
+   - *DEPO has repeatedly heard from Veteran participants that it is difficult to complete forms online due to certain conditions, which contributes to frustration when trying to complete tasks such as online scheduling*
+   - Users have information overload on VA appointments page
+   - Users have information overload on VA schedule an appointment page
+   - Users have information overload/confusion as they are going through the flow to schedule an appointment
+     - Users are looking for, but can't find a specific appointment type, either because it is not listed/supported by online scheduling, at that location, or the user has a different name for the appointment type than what is listed
+     - Users are looking for a specific location, but can't find the specific location they have in mind, either because the location isn't listed or it is listed using a different name/nested within a category they are unfamiliar with
+     - Users are looking for a specific provider, but can't find the specific provider they have in mind
+   - Users are confused by the different look/feel, if this is new to them. Easily give up, rather than trying to figure it out 	
+
 ## Method	
 1.	What method of research are you planning? 	
-  - **Sample Methodologies**: Directed interviews; ethnographic interviews; observation; card sorts; tree tests; guerilla usability testing; usability testing with low- or high-fidelity prototypes; user acceptance testing. 	
-  - **Also mention:** in-person moderated, remote moderated, remote unmoderated	
-  	
-2.	Why this method? How does this methodology help you answer your research questions? 	
+   - Remote moderated usability research and usability testing mid-fidelity prototypes
+2.	Why this method? How does this methodology help you answer your research questions? 
+   -  
 3.	Where are you planning to do your research? *If in person, mention the location, point of contact, arrangements, etc. If online, mention which tool you'll be using (GTM, Join.me, etc.)*	
-4.	What will you be testing? *(Design mocks, card sort, prototype, page, content, etc.)* 	
-5.  If remote: What tool do you plan to use (Zoom, GoToMeeting, Webex)	
+   - Zoom
+4.	What will you be testing? *(Design mocks, card sort, prototype, page, content, etc.)* 
+   - https://www.va.gov/health-care/schedule-view-va-appointments/
```

Comment:
The actual url for the tool is - www.va.gov/health-care/schedule-view-va-appointments/appointments/
But the one you have is also relevant if we are starting from that page

Reply:
When we talked last week, I thought we said we wanted to get feedback on this page to answer `[9 Login and tool instructions] Have you seen this before? What do you think?`

It's possible I copied the wrong link from our convo. just let me know and we can swap it for the right one.

Reply:
we're going to stick with the landing page to get feedback on instructions